Sources/Sentry/SentryTracer.m

```diff
@@ -188,8 +189,9 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
 #endif // SENTRY_HAS_UIKIT
 
 #if SENTRY_TARGET_PROFILING_SUPPORTED
-    if (_configuration.profilesSamplerDecision.decision == kSentrySampleDecisionYes
-        || isTracingAppLaunch) {
+    if (!hub.client.options.enableContinuousProfiling
```

Comment:
`m`: A test or updating an existing test for this new check would be great.

Reply:
This is true. I think this logical change would be more appropriate in a later branch where I'm actually changing implementation, which will include tests, so I'll move it there.